package NAlice;

message TPersonalIntentsRecord {
    message TPersonalIntents {
        message TIntentCount {
            optional string Intent = 1; // Can be used both as intent and condition hash
            optional uint64 Count = 2;
            optional uint64 LastUsedTime = 3;
        }

        // Intent usage stats
        repeated TIntentCount Intents = 1;
        // Applied SuccessCondition stats
        repeated TIntentCount PostrollConversions = 2;
    }

    optional string Url = 1;
    optional TPersonalIntents PersonalIntents = 2;
}
